ophyd/controls/signal.py

```diff
@@ -195,6 +207,17 @@ def __init__(self, read_pv, write_pv=None,
         self._pv_kw = pv_kw
         self._auto_monitor = auto_monitor
 
+        if dtype is None:
+            dtype = float
+        valid_dtypes = ['float', 'int', 'str']
```

Comment:
This is the wrong list of valid entries for dtypes.  See http://nsls-ii.github.io/arch/metadatastore-format.html#schema